### PR TITLE
Development helmchart fix

### PR DESCRIPTION
**Pull request by waleedhammam**
Screenshot from 2022-01-17 13-37-56

### CODE REVIEW

File: ygg_entrypoint.sh [hunk 1]
```diff
@@ -2,16 +2,13 @@
...
-mknod /dev/net/tun c 10 200
-chmod 600 /dev/net/tun
-
-if [ ! -f "/etc/config.conf" ]; then
-  echo "generate /etc/config.conf"
-  yggdrasil --genconf > "/etc/config.conf"
+if [ ! -f "/etc/yggdrasil.conf" ]; then
+  echo "generating new configurations at /etc/yggdrasil.conf"
+  yggdrasil --genconf > "/etc/yggdrasil.conf"
+  sed -i "/Peers: \[\]/c\  Peers: \n  [\n    tls:\/\/54.37.137.221:11129\n  ]" /etc/yggdrasil.conf
```
Comment (OmarElawady) [inline ygg_entrypoint.sh]: peers come and go, maybe a list of active peers can be generated instead from the public peers list. Or they can be passed as an env var to the chart.

File: ygg_sample.conf [hunk 2]
```diff
@@ -0,0 +1,81 @@
+{
+  # List of connection strings for outbound peer connections in URI format,
+  # e.g. tls://a.b.c.d:e or socks://a.b.c.d:e/f.g.h.i:j. These connections
+  # will obey the operating system routing table, therefore you should
+  # use this section when you may connect via different interfaces.
+  Peers: [
+    tls://54.37.137.221:11129
```
Comment (dmahmouali) [inline ygg_sample.conf:7]: what about increasing the number of peers to cover each other?

the online peers: https://publicpeers.neilalexander.dev

File: ygg_entrypoint.sh (hunk 1 above)
Comment (waleedhammam) [reply ygg_entrypoint.sh]: done as env variable

File: ygg_sample.conf (hunk 2 above)
Comment (waleedhammam) [reply ygg_sample.conf:7]: done sir, check readme

**Review by dmahmouali — APPROVED** [review]
LGTM

**Review by OmarElawady — APPROVED** [review]

**Pull request merged**